workspace/linkedService/Mipins_Dev1.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "Mipins_Dev1",
+	"type": "Microsoft.Synapse/workspaces/linkedservices",
+	"properties": {
+		"annotations": [],
+		"type": "SqlServer",
+		"typeProperties": {
+			"server": "pins-prod-pdac-sql.database.windows.net",
+			"database": "MiPINS-DEV-ISS",
+			"encrypt": "mandatory",
+			"trustServerCertificate": false,
+			"authenticationType": "SQL",
+			"userName": "bi_developer",
+			"encryptedCredential": "ew0KICAiVmVyc2lvbiI6ICIyMDE3LTExLTMwIiwNCiAgIlByb3RlY3Rpb25Nb2RlIjogIktleSIsDQogICJTZWNyZXRDb250ZW50VHlwZSI6ICJQbGFpbnRleHQiLA0KICAiQ3JlZGVudGlhbElkIjogIlNZTkFQU0VAQUVFRDhDRUEtMjg4OC00MjVBLUJDNEEtN0QzQ0JCQTMzQTBFXzU5YWMxOGUxLTgyNWEtNDkzOS1iM2Y5LTQxNTUzYjlmOTRlYSINCn0="
```

Comment:
Why do we need a new linked service? Do we already not have ls_sql_mipins_dev?

Reply:
Also, is this a secret value directly added to the code rather than storing in keyvault?

Reply:
Mipins_Dev1 not required It is created other SQL DB  data set connection, I will change now.

Reply:
Azure secrets will be picked up by Github security secret scanning but I don't know if that covers everything. So I don't think this is an actual secret. However, linked services should be kept to a minimum and we seem to have duplicates. The majority should just point to keyvault, using ls_kv, and selecting the secret from the keyvault. This ensures consistency and simplicity.